### PR TITLE
mstrok.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
See also `w.mstrok.ru` and `ww.mstrok.ru`

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/mstrok.ru.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+hackertrace1.internal.nsa.gov.mstrok.ru ¹
+hackertrace2.internal.nsa.gov.mstrok.ru ¹
+hackertrace3.internal.nsa.gov.mstrok.ru ¹
+hackertrace4.internal.nsa.gov.mstrok.ru ¹
+w.mstrok.ru ¹
```
Comment (J0WI) [inline src/chrome/content/rules/mstrok.ru.xml]: + ww.mstrok.ru

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**